WcaOnRails/app/views/panel/index.html.erb

```diff
@@ -47,9 +47,24 @@
       <li>
         <%= link_to "Upcoming competitions for your subordinate Delegates", competitions_for_senior_path %>
       </li>
+      <li>
+        <%= link_to "Modify Delegates on Probation", edit_team_path(Team.probation) %>
+      </li>
     </ul>
   <% end %>
 
+
+  <% if current_user.team_leader?(Team.wfc) || current_user.team_senior_member?(Team.wfc) %>
```

Comment:
This duplicates the logic you added to `can_edit_team?` above. Did you intentionally leave Senior Delegates out here? How about changing this to `current_user&.can_edit_team?(Team.probation)`?

Reply:
Yes! The panel is split of into sections of position. So Senior Delegates have a section and WFC has a section. 

So this would appear twice if someone was a Senior Delegate and a WFC Senior Member.

I'd prefer to keep it how it is as its what I think Alberto or Bob also agreed with.

Reply:
If the WFC section would ever have more added to it, then the if statment would become for all WFC members and then the subsection would be based on if they can edit the probationary team. However since for regular WFC members, it would be blank, I think the way it is now is best.